src/jit/ee_il_dll.cpp

```diff
@@ -163,6 +163,20 @@ CorJitResult CILJit::compileMethod (
         return g_realJitCompiler->compileMethod(compHnd, methodInfo, flags, entryAddress, nativeSizeOfCode);
     }
 
+    if (!JitConfig.isInitialized())
```

Comment:
Okay.

FWIW, this is roughly as thread-safe as the existing code, which may also initialize config values from multiple threads simultaneously. The likelihood of a race in the existing code might be smaller since initialization is distributed throughout the codebase, but it is still there.